src/graph/executor/query/RollUpApplyExecutor.cpp

```diff
@@ -114,14 +122,25 @@ DataSet RollUpApplyExecutor::probeSingleKey(Expression* probeKey,
 DataSet RollUpApplyExecutor::probe(std::vector<Expression*> probeKeys,
                                    Iterator* probeIter,
                                    const std::unordered_map<List, List>& hashTable) {
+  std::vector<Expression*> probeKeysCopy;
```

Comment:
`singleKeyProbe` and `singleKeyBuild` also need.

Reply:
The expr was cloned when pass to the function:
`    result = probeSingleKey(rollUpApplyNode->compareCols()[0]->clone(), lhsIter_.get(), hashTable);
`